docs/tutorial/visualization/non_blocking_visualization.rst

```diff
@@ -72,8 +70,8 @@ Transform geometry and visualize it
 
 .. literalinclude:: ../../../examples/python/visualization/non_blocking_visualization.py
    :language: python
-   :lineno-start: 59
-   :lines: 59-72
+   :lineno-start: 37
+   :lines: 37-49
```

Comment:
@ssheorey These changes are taken care of here - https://github.com/isl-org/Open3D/pull/6321/files#diff-4d3bb3b76c894acd5303dbc1d2a0d919882866be425e1ce6279c4e59c0e951c0R28 (reminding just in case).